src/main/java/com/minecraftabnormals/abnormals_core/core/util/BiomeUtil.java

```diff
@@ -69,6 +79,46 @@ public static synchronized void addEndBiome(RegistryKey<Biome> key, int weight)
 	public static synchronized void markEndBiomeCustomMusic(ResourceLocation biomeName) {
 		CUSTOM_END_MUSIC_BIOMES.add(biomeName);
 	}
+	
+	/**
+	 * Adds an ocean biome to generate with a given weight.
+	 * <p>This method is safe to call during parallel mod loading.</p>
+	 *
+	 * @param type The {@link OceanType} to register the {@link Biome} to.
+	 * @param biome A {@link Biome} {@link RegistryKey} to add.
+	 * @param weight The weight for the {@link Biome}.
+	 */
+	public static synchronized void addOceanBiome(OceanType type, RegistryKey<Biome> biome, int weight) {
+		addOceanBiome(type, biome, null, weight);
+	}
+	
```

Comment:
In my opinion, this method isn't needed and adds clutter.
I don't think people will mind having to pass `null` into the `deep` parameter.